File: vue/src/store/aa01.ts
import { defineStore } from 'pinia'

import {deleteAa01, getAa01List, writeAa01, Page} from "../api";

export interface aa01Type {
    gubun: String
    useyn: String
    sortOrder: Number
    attr: String
    cname: String
    convType: String
    memo: String
    addUser?: String
    addDate?: Date
    addIp?: String
    updateUser?: String
    updateDate?: Date
    updateIp?: String
    bigo: String
}
export const Aa01Store = defineStore("Aa01Store",
    {
        state: () => ({
            aa01List:[] as aa01Type[],
        }),
        getters: {


        },
        actions: {
            getAa01List(params : Page) {
                return getAa01List(params)
            },
            delete(gubn : String[]) {
                return deleteAa01({gubn:gubn})
            },
            write(params: aa01Type)
            {
                return writeAa01(params)
            }

        }
    })

File: src/main/java/com/dipol/pdfconvert/data/entity/Aa01.java
package com.dipol.pdfconvert.data.entity;

import lombok.Data;

import javax.persistence.*;
import java.sql.Timestamp;
import java.util.Date;

@Entity
@Table(name = "aa01", schema = "pdftest")
@Data
public class Aa01 {
    @Id
    @Column(name = "gubn")
    private String gubn;
    @Basic
    @Column(name = "useyn")
    private String useyn;
    @Basic
    @Column(name = "sort_order")
    private Integer sortOrder;
    @Basic
    @Column(name = "attr")
    private String attr;
    @Basic
    @Column(name = "cname")
    private String cname;
    @Basic
    @Column(name = "conv_type")
    private String convType;
    @Basic
    @Column(name = "memo")
    private String memo;
    @Basic
    @Column(name = "add_user")
    private String addUser;
    @Basic
    @Column(name = "add_date")
    private Date addDate;
    @Basic
    @Column(name = "add_ip")
    private String addIp;
    @Basic
    @Column(name = "update_user")
    private String updateUser;
    @Basic
    @Column(name = "update_date")
    private Date updateDate;
    @Basic
    @Column(name = "update_ip")
    private String updateIp;
    @Basic
    @Column(name = "bigo")
    private String bigo;

}
